api/tests/team/unit/scripts/send-organization-invitations-script_test.js

```diff
@@ -21,6 +23,21 @@ describe('unit | team | scripts | send-organization-invitations-script', functio
         { 'Organization ID': 1, email: 'test@example.net', locale: 'fr', role: 'MEMBER' },
       ]);
     });
+
+    it('Refuses with an incorrect email', async function () {
```

Comment:
On met les texte des tests en minuscule :

```suggestion
    it('refuses with an incorrect email', async function () {
```